Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml

```diff
@@ -0,0 +1,59 @@
+Parser:
```

Comment:
please update this as well:
https://github.com/Azure/Azure-Sentinel/blob/c181e4c0ebc237648a9925916fc7bb15458e2b5b/Parsers/ASimProcessEvent/Parsers/ASimProcessEventTerminate.yaml

Reply:
done

Reply:
changes not visible, please re-check

Reply:
added